angular.json

```diff
@@ -0,0 +1,157 @@
+{
+  "$schema": "./node_modules/@angular/cli/lib/config/schema.json",
+  "version": 1,
+  "newProjectRoot": "projects",
+  "projects": {
+    "mobile-content-admin": {
+      "root": "",
+      "sourceRoot": "src",
+      "projectType": "application",
+      "architect": {
+        "build": {
+          "builder": "@angular-devkit/build-angular:browser",
+          "options": {
+            "outputPath": "dist",
+            "index": "src/index.html",
+            "main": "src/main.ts",
+            "tsConfig": "src/tsconfig.app.json",
+            "polyfills": "src/polyfills.ts",
+            "assets": [
+              "src/assets",
+              "src/favicon.ico"
+            ],
+            "styles": [
+              "src/styles.css"
+            ],
+            "scripts": [
+              "node_modules/ace-builds/src-min/ace.js"
```

Comment:
You manually add this? Does this give you a newer version? If you didn't do this, that's an oddly specific import...

Reply:
yeah, I had to manually add it, otherwise I get an error from the build. the `ng2-ace-editor` module is pretty flaky right now. I'm going to switch to the newer one once I get to Angular 9